storage/s3_storage.py

```diff
@@ -40,13 +40,19 @@ def save_to_filename(self, file_path: str) -> None:
         client = self._connect()
 
         transfer = boto3.s3.transfer.S3Transfer(client)
-        transfer.download_file(self._bucket, self._keyname, file_path)
+        try:
+            transfer.download_file(self._bucket, self._keyname, file_path)
+        except IOError:
```

Comment:
Is `IOError` correct? That's a built-in Python exception, so I would expect it to only be raised if there's something wrong locally, not if the file is missing in the S3 bucket.